fatiando/tests/mesher/test_oblate_ellipsoid.py

```diff
@@ -0,0 +1,49 @@
+from ...mesher import OblateEllipsoid
+import numpy as np
+from numpy.random import rand
+
+
+def test_oblate_ellipsoid_copy():
+    orig = OblateEllipsoid(1, 2, 3, 4, 6, 10, 30, {
+        'remanence': [10000., 25., 40.],
+        'k': [0.562, 0.485, 0.25, 90., 0., 0.]})
+    cp = orig.copy()
+    assert orig is not cp
+    assert orig.x == cp.x
+    assert orig.y == cp.y
+    assert orig.z == cp.z
+    assert orig.a == cp.a
+    assert orig.b == cp.b
+    assert orig.props == cp.props
+    assert np.array_equal(orig.V(), cp.V())
+
+    cp.x = 4
+    cp.y = 6
+    cp.z = 7
+    cp.props['remanence'] = [100., -40., -25.]
+    assert orig.x != cp.x
+    assert orig.y != cp.y
+    assert orig.z != cp.z
+    assert orig.props['remanence'] != cp.props['remanence']
+
+
+def test_oblate_ellipsoid_V_orthogonal():
+    a = 1000.*rand()
```

Comment:
Never test using random values! There is no way to know what values causes the tests to fail and no way to reproduce the failing test. Use a specific value. If you want to test several values, use a list and a `for` loop.